bson/bsoncodec/default_value_encoders.go

```diff
@@ -462,6 +440,19 @@ func (dve DefaultValueEncoders) SliceEncodeValue(ec EncodeContext, vw bsonrw.Val
 	return aw.WriteArrayEnd()
 }
 
+func (dve DefaultValueEncoders) lookupElementEncoder(ec EncodeContext, origEncoder ValueEncoder, currVal reflect.Value) (ValueEncoder, reflect.Value, error) {
+	if origEncoder != nil || (currVal.Kind() != reflect.Interface) {
+		return origEncoder, currVal, nil
+	}
+	currVal = currVal.Elem()
+	if !currVal.IsValid() {
+		return nil, currVal, fmt.Errorf("Cannot encode invalid element")
```

Comment:
nit: per https://github.com/golang/go/wiki/CodeReviewComments#error-strings, error strings shouldn't be capitalized.

Reply:
done